### ISSUE TITLE
Incompatiblity with Kitty terminal

### ISSUE THREAD

**Issue by kbd** [austinjones/tab-rs#337]
I'm in the process of switching from iTerm2 to Kitty. Kitty sets `$TERM` to `xterm-kitty`. When I run `tab` within Kitty, I get:

![Screen Shot 2021-01-31 at 2 37 09 AM](https://user-images.githubusercontent.com/728060/106380108-585ce480-636d-11eb-89ca-832e7b5054be.png)

```shell
$ TERM=xterm-256color tab foo
```

Seems to work as expected (so far?).

**Comment by austinjones** [comment]
Interesting.  Tab passes through the $TERM env var.  Is kitty setting the $TERMINFO env var?  I might need to pass that along as well.

**Comment by kbd** [comment]
> Is kitty setting the $TERMINFO env var?

Sorry for slow response. Yeah it is:

```
$ echo $TERM     
xterm-kitty
$ echo $TERMINFO 
/Applications/kitty.app/Contents/Resources/kitty/terminfo
$ cd /Applications/kitty.app/Contents/Resources/kitty/terminfo
$ tree                        
.
└── 78
    └── xterm-kitty
```

**Comment by austinjones** [comment]
I just pushed a fix in [v0.5.7](https://github.com/austinjones/tab-rs/releases/tag/v0.5.7).  I saw a similar 'unknown terminal' message in fish and the same $TERM/$TERMINFO vars.  It should work fine now!
